pytest_html/plugin.py

```diff
@@ -167,21 +166,17 @@ def __lt__(self, other):
         def create_asset(
             self, content, extra_index, test_index, file_extension, mode="w"
         ):
-
-            hash_key = "".join([self.test_id, str(extra_index), str(test_index)])
-            hash_generator = hashlib.md5()
-            hash_generator.update(hash_key.encode("utf-8"))
-            hex_digest = hash_generator.hexdigest()
-            # 255 is the common max filename length on various filesystems,
-            # we subtract hash length, file extension length and 2 more
-            # characters for the underscore and dot
-            max_length = 255 - len(hex_digest) - len(file_extension) - 2
-            asset_file_name = "{0}_{1}.{2}".format(
-                hash_key[:max_length], hex_digest, file_extension
-            )
+            # 255 is the common max filename length on various filesystems
+            asset_file_name = "{}_{}_{}.{}".format(
+                self.test_id.replace(":", "_"),
```

Comment:
Are you confident that `test_id` can't contain other unsafe characters?

Reply:
Now I am. (see the revised code) 😊 @davehunt 

Now **everything** that isn't `a-zA-Z0-9` or `_` or `.` is replaced with `_`